seedlet/classes/class-seedlet-custom-colors.php

```diff
@@ -263,7 +279,9 @@ function seedlet_generate_custom_color_variables( $context = null ) {
 	 */
 	function seedlet_custom_color_variables() {
```

Comment:
Unrelated to this PR, but I think this function should be renamed to be more specific to what it does, since there is already a private variable of this class called `seedlet_custom_color_variables`. 

```suggestion
	function seedlet_enqueue_custom_color_variables() {
```

Reply:
Agreed. 

I kind of lean towards just doing it in this PR unless there’s a good reason not to.